server/cmd/mmctl/client/client.go

```diff
@@ -159,4 +159,5 @@ type Client interface {
 	GetPreferenceByCategoryAndName(ctx context.Context, userId, category, preferenceName string) (*model.Preference, *model.Response, error)
 	UpdatePreferences(ctx context.Context, userId string, preferences model.Preferences) (*model.Response, error)
 	DeletePreferences(ctx context.Context, userId string, preferences model.Preferences) (*model.Response, error)
+	GetOrCreateDirectChannel( ctx context.Context, otherUserID string) (*model.Channel, *model.Response, error)
```

Comment:
Hello @hanzei 
I added the GetOrCreateDirectChannel to the mmctl `client` interface since the method was not there